examples/wait-for-iframe-load/README.md

```diff
@@ -0,0 +1,18 @@
+# Wait For Iframe Loading
+
+**Test Code**: [wait-for-iframe-load.js](wait-for-iframe-load.js)
+
+TestCafe implements [wait mechanisms](https://devexpress.github.io/testcafe/documentation/guides/concepts/built-in-wait-mechanisms.html) that automatically suspend the test until all required page elements are loaded and ready for interaction. These mechanisms apply to page loads, animations, XHR requests, iframe initializations, etc. TestCafe waits until the target element is ready, or reports an error if this does not happen within a timeout.
```

Comment:
This example demonstrates only the call `swithToIframe` method with a parameter.
It's better to add this example in the corresponding [help topic](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/switchtoiframe.html).